use-cases/using-SRIOV-and-kata.md

```diff
@@ -0,0 +1,313 @@
+# Setup to use SR-IOV with Kata Containers
+
+Single Root I/O Virtualization (SR-IOV) enables splitting a physical device into
+virtual functions (VFs). Virtual functions enable direct passthrough to virtual
+machines or containers. For Kata Containers, we enabled a Container Network 
+Model (CNM) plugin. Additionally, we made the necessary changes in the 
+runtime to detect virtual functions in a container's network namespace to be 
+able to use SR-IOV for network based devices.
+
+## Install the SR-IOV Docker\* plugin
+
+To create a network with associated VFs, which can be passed to
+Kata Containers, you must install a SR-IOV Docker plugin. The 
+created network is based on a physical function (PF) device. The network can
+create 'n' containers, where 'n' is the number of VFs associated with the 
+Physical Function (PF).
+
+To install the plugin, follow the [plugin installation instructions](https://github.com/clearcontainers/sriov).
+
+
+## Host setup for SR-IOV
+
+In order to setup your host for SR-IOV, the following has to be true:
+
+- The host system must support Intel VT-d. 
+- Your device (NIC) must support SR-IOV
+- The host kernel must have Input-Output Memory Management Unit (IOMMU) 
+  and Virtual Function I/O (VFIO) support.
+- `CONFIG_VFIO_NOIOMMU` must to be disabled in the host kernel
+  configuration. You must rebuild your host system's kernel in
+  order to disable `CONFIG_VFIO_NOIOMMU` in the kernel configuration.
+- Optionally, you might need to add a PCI override for your Network Interface 
+  Controller (NIC). The immediately following section describes how to assess 
```

Comment:
Can we change the 'the immediately following' reference to a hotlink to the correct section - that way it will follow it and stay upto date (I suspect it might be out of date already). thx